### DISCUSSION TITLE
Feature Request: Mute daily.dev

### DISCUSSION THREAD

**Discussion by mrkosima** [dailydotdev/daily#1462] [Feature Requests]
Hi there 👋,
I really enjoy the daily.dev community and the Chrome Extension!

**[Feedback]:** On days when I need to focus on work, I find the Dev UI can sometimes become distracting, and it feels more like background noise rather than a source of interest.

As a less active user, I would appreciate an option to mute or pause content. Here are some ideas:
**[Feature Suggestion]**: 

-  Mute by time window, e.g. `09:00 - 16:00`
- Mute by schedule, e.g., `Mon, Tue, Thu`
- Mute temporarily, e.g., `for 2 hours`
- Turn off the app, but keep the button to turn it back on when needed
- Reduce the number of cards shown, perhaps to a single one that might be the "best from last 1 hour"

What do you think?

**Comment by Programazing** [comment]
I completely missed this when I made #1465, and I agree.

**Comment by nimrodkra** [comment]
@mrkosima have you considered using the "pause new tab" feature?
![Screenshot 2024-09-29 at 8 22 34](https://github.com/user-attachments/assets/05094b18-71d5-4ba1-9cfe-a06763f021ec)
![Screenshot 2024-09-29 at 8 22 39](https://github.com/user-attachments/assets/39cd350b-7ce5-4d0c-b627-bc3014ba6b88)

**Reply by mrkosima** [reply]
@nimrodkra , oh, I missed this feature! Nice, it is already helpful for my issue!

Do you think it would be beneficial to extend the functionality to be more intelligent, e.g. with the options from the topic message?

**Comment by nimrodkra** [comment]
We could improve it a little bit more but the current adoption is too low
to justify putting it in priority so unfortunately we can't commit to any
improvements in the near future.
